org.opent2t.sample.hub.superpopular/com.wink.hub/js/tests/test.js

```diff
@@ -0,0 +1,29 @@
+var test = require('ava');
+var OpenT2T = require('opent2t').OpenT2T;
+var config = require('./testConfig');
+
+console.log(config);
```

Comment:
Do we need to JSON stringify? This just prints out as [object, object] for me... don't know why? Recommend you use the commented out line below or modify it ... might want to do JSON.stringify(foo, null, 2) to make it pretty per earlier suggestion by @begillil.